daml-script/export/integration-tests/reproduces-transactions/test/scala/com/daml/script/export/ReproducesTransactions.scala

```diff
@@ -194,12 +196,18 @@ trait ReproducesTransactions
     )
   } yield ()
 
+  private def uploadDarFiles(client: LedgerClient) =
+    Future.sequence(packageFiles.map { dar =>
+      client.packageManagementClient.uploadDarFile(ByteString.readFrom(new FileInputStream(dar)))
+    })
```

Comment:
I think this belongs on `AbstractSandboxFixture`, where many (most?) of the expansions in this PR can use it.